sachungo/movies
# Movie info update

### Which issue does this PR relate to?

https://www.pivotaltracker.com/story/show/160449833

### What are the changes implemented in the PR?



- Updated the info page UI
   - Added navigation bar
   - Formatted the date of release
   - Added comma to genres
   - Applied the max width to the page
   - Added CSS transitions
   - Added a button on empty/error message for navigating back to the homepage
### How should the implementation be QA'd?



- Access the review app by clicking the deployment button in the commenst below
- Verify that you can see the navigation bar, both on desktop and mobile web
- Verify that all still works well for the movie info page

### Anything else to note


N/A

### Screenshots and/or gifs


*Desktop*
<img width="1225" alt="image 2018-09-14 at 6 04 21 am" src="https://user-images.githubusercontent.com/12892171/45527421-25d5df80-b7e4-11e8-980b-615cfd6e4a4b.png">

---

*Mobile*
![image 2018-09-13 at 3 16 32 pm](https://user-images.githubusercontent.com/12892171/45527433-35edbf00-b7e4-11e8-922e-a7533f297b75.png)
